### PR TITLE
New Sitemap and User Switch Functionality

### PR DESCRIPTION
**Pull request by RobLoweMOJ**

### CODE REVIEW

**Review by brown-a2 — APPROVED** [review]

File: component/Sitemap/SitemapSettings.php [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+class SitemapSettings extends Sitemap
+{
+    public $helper;
+
+    public function __construct()
+    {
+        global $mojHelper;
+        $this->helper = $mojHelper;
+
+        //parent::__construct();
```
Comment (brown-a2) [inline component/Sitemap/SitemapSettings.php]: is this supposed to be commented out?

File: component/Users/assets/css/main.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
+#wpadminbar .quicklinks #wp-admin-bar-tikemp_impresonate_user ul li .ab-item {
```
Comment (brown-a2) [inline component/Users/assets/css/main.css:1]: At some point we should use SASS in our plugins?

**Review by wilson1000-MoJ — APPROVED** [review]
Looks good buddy! Do you think we should load dev-master in our local env. before creating a release?

File: component/Sitemap/SitemapSettings.php (hunk 1 above)
Comment (RobLoweMOJ) [reply component/Sitemap/SitemapSettings.php]: Removed 👍

File: component/Users/assets/css/main.css (hunk 2 above)
Comment (RobLoweMOJ) [reply component/Users/assets/css/main.css:1]: We should. I just copied this from the prev plugin but we should look at putting sass into components plugin

**Pull request merged**